etc/schema.json

```diff
@@ -331,6 +331,40 @@
             },
             "additionalProperties": false
         },
+        "arp": {
+            "type": "object",
+            "optional": true,
+            "properties": {
+                "hw_type": {
+                    "type": "string"
+                },
```

Comment:
I'm undecided if this should be a number or a string.  As a "dashboard" builder I would need to know the full set of strings we used if I was hunting for something instead of the number which is fixed.

Thoughts?  Ping @pevma, @jlucovsky as well on this.